docs/platform_concepts/growth/dataexplorer/customtables.md

```diff
@@ -67,7 +67,9 @@ You can filter data by a single column value or a combination of column values.
 
 ### Save filtered data as reports 
 
-Once you have applied filters, a button labeled **Save query** will appear. Click on this button to save the filtered data as a saved report. This report can then be accessed under the **Saved reports** section every time you open the data explorer, without the need to repeat all the filtering steps again.
+Once you have applied filters, a button labeled **Save query** will appear. Click on this button to save the filtered data as a saved report. This report can then be accessed under the **Saved reports** section every time you open the data explorer, without the need to repeat all the filtering steps again.             
```

Comment:
Just a small suggestion for better readability -
-Once you have applied filters, a button labeled **Save query** will appear. Click this button to save the filtered data as a report, which can be accessed under the **Saved reports** section whenever you open the data explorer. This way, you can retrieve the report without having to repeat the filtering steps again.